src/Compilers/CSharp/Test/Syntax/Parsing/ExtensionParsingTests.cs

```diff
@@ -73,107 +73,46 @@ public void ExtensionParsing(bool isExplicit)
         EOF();
 
         UsingTree(text, options: TestOptions.Regular11,
```

Comment:
>Regular11 [](http://example.com/codeflow?start=45&length=9)

@jcouv It looks like we should switch to `C# 12` here and in other similar places. I am not suggesting to do this in context of this PR.